### PR TITLE
Use REST API to deploy processes to model

### PR DESCRIPTION
**Pull request by AaronWantoch**

### CODE REVIEW

File: src/model_manager.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import json
+import requests
```
Comment (Lenetis) [inline src/model_manager.py:2]: Please add requests to requirements.txt

File: src/model_manager.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            response_dict = json.loads(response.text)
+            self.process_instance_id = response_dict['id']
+
+    def delete_process(self):
+        if self.useModel and self.process_instance_id is not None:
+            requests.delete(self.modelURL+"/process-instance/"+self.process_instance_id)
+            self.process_instance_id = None
+
+    
+    def complete_user_task(self, data={}, end=False):
```
Comment (Lenetis) [inline src/model_manager.py]: `data={}` - default value in Python should not be mutable. I'd suggest changing this to something like:
```
def complete_user_task(self, data=None, end=False):
    if data is None:
        data = {}
    # rest of the method
```

Same thing with `complete_service_task()`

File: src/model_manager.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                "workerId": worker_id,
+                "maxTasks": 1,
+                "topics": [
+                    {
+                    "topicName": topic_name,
+                    "lockDuration": 600000
+                    }
+                ]
+            }
+            requests.post(self.modelURL+"/external-task/fetchAndLock", json=data).text
```
Comment (Lenetis) [inline src/model_manager.py]: Why the `.text` after requests.post()? It doesn't seem to do anything

File: src/model_manager.py
Comment (Lenetis) [inline src/model_manager.py:1]: Please add docstrings and type hinting to ModelManager methods

**Review by bstrzelecki — APPROVED** [review]

**Review by Lenetis — APPROVED** [review]

**Pull request merged**